lib/healthcheck.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * create the healthcheck handler
+ *
+ * @param {Object} keystone - keystone instance
+ * @returns {function} - a healthcheck handler to check the health status of keystone(e.g. db connection)
+ */
+function createHealthCheckHandler(keystone){
+  return function(){
+    var readyState = keystone.mongoose.connection.readyState
```

Comment:
```suggestion
    var readyState = keystone?.mongoose?.connection?.readyState
```

Reply:
- avoid js error if keystone | mongoose | connection is undefined.
- maybe you can also add undefined check for readyState value